From ea557350f1873e6905e6c4d3b28c042bacfd63cd Mon Sep 17 00:00:00 2001
From: Kevin Zhang <k.zhang@freescale.com>
Date: Wed, 19 Mar 2008 19:56:14 -0500
Subject: [PATCH] ENGR00069345 Fix kernel random crash due to NAND driver for 4K page NAND

This change was missed after initial Linux bringup work. It increased the
NAND default page size and spare size in order to support 4K page NAND.

Signed-off-by: Kevin Zhang <k.zhang@freescale.com>
---
 include/linux/mtd/nand.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/mtd/nand.h b/include/linux/mtd/nand.h
index c42bc7f..a830fd6 100644
--- a/include/linux/mtd/nand.h
+++ b/include/linux/mtd/nand.h
@@ -45,8 +45,8 @@ extern void nand_wait_ready(struct mtd_info *mtd);
  * is supported now. If you add a chip with bigger oobsize/page
  * adjust this accordingly.
  */
-#define NAND_MAX_OOBSIZE	64
-#define NAND_MAX_PAGESIZE	2048
+#define NAND_MAX_OOBSIZE	218
+#define NAND_MAX_PAGESIZE	4096
 
 /*
  * Constants for hardware specific CLE/ALE/NCE function
-- 
1.5.4.4

